tutorials/fundamentals/primitives.md

```diff
@@ -0,0 +1,56 @@
+Javascript has seven primitive data types:
+
+- string
+- number
+- bigint
+- boolean
+- undefined
+- symbol
+- null
+
+What makes these primitives is that they are not objects,
+they do not have any methods, and they are immutable,
```

Comment:
None of these are entirely accurate. While you might technically say primitives are immutable in JavaScript, immutable is commonly used to mean something different in JavaScript. Also, you can call methods on primitives. For example, `(42).toString() === '42'`. I'd just say that "In JavaScript, a primitive is any value that isn't an object."

I know MDN says some of these things, and, while they may be technically correct, they're misleading and irrelevant in practice.